### PR TITLE
Added Tableau 8

### PR DESCRIPTION
**Pull request by mattnelson**

### CODE REVIEW

File: Casks/tableau8.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'tableau8' do
+  version '8.2.9'
+  sha256 '2644c7b9fd6c453d00f6641ddb681b87ba5cdfa16c6d840163295fc176f54ee1'
+
+  url 'https://downloads.tableau.com/esdalt/8.2.9/TableauDesktop.dmg'
```
Comment (fanquake) [inline Casks/tableau8.rb]: You can use #{version} in the URL here

File: Casks/tableau8.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Tableau'
+  homepage 'http://www.tableausoftware.com/'
```
Comment (fanquake) [inline Casks/tableau8.rb]: It looks like there's a https link available, can you use that instead.

**Comment by mattnelson** [comment]
Addressed comments in 7b14e98209a198f724aab61e75fe6fae435f4b69

Also created separate casks for the 8.2 and 8.3 streams.

**Comment by mattnelson** [comment]
Anything else holding up this pull?

**Comment by nanoxd** [comment]
Sorry about that. Looks good!

**Pull request merged**